app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Manage Dev Portal Teams
+content_type: how-to
+---
+
+A common scenario for a Dev Portal is to restrict developers' ability to view or request access to specific services based on their permissions.
+
+{{site.konnect_short_name}} Portal enables administrators with the ability to define Role-Based Access Control (RBAC) for teams of developers through the Konnect API.
+
+Portal RBAC supports two different roles for Services that can be applied to a Team of developers:
+
+* API Viewer: provides read access to the documentation associated with a Service.
+* API Consumer: can register applications to consume versions of a Service.
+
+You can use the API to create a team, assign a role to the team, and finally, add developers to the team.
+
+In this guide, you will set up two developer teams (more info about the scenario) and then enable Portal RBAC using a hypothetical scenario.
+
+## Prerequisites
+* Two [test developer accounts registered](/konnect/dev-portal/dev-reg/)
+* A [runtime configured](/konnect/getting-started/configure-runtime)
+
+## Configure developer teams 
+
+In this scenario, you are a product manager at a pizza company who is in charge of their online app. You're asked to create a Dev Portal that allows delivery companies to use their APIs to deliver the company's pizzas. However, you want to make sure only trusted delivery partners can build apps using your APIs and deliver your pizzas. 
+
+You've decided to create two groups of developers with different access levels to your APIs: 
+* Delivery Partners: This developer team will be able to view and consume your APIs so they can use them in their own apps to deliver your pizzas.
+* New Partners: This developer team will contain delivery groups that the company is still in the process of vetting and you don't want to allow them to consume your APIs in their apps yet. You give this group view-only privileges so they can see what your API specs look like. 
+
+{:.important}
+> **Important:** If you currently have developers in your {{site.konnect_short_name}}, you must configure developer teams before enabling Portal RBAC (which is disabled by default). If you enable Portal RBAC before configuring teams for your developers, it will prevent all developers from seeing any services in your Dev Portal. We recommend setting up your teams and permissions before enabling RBAC which will allow for a seamless transition: developers see what they're supposed to, instead of nothing at all.
+
+### Create an API Product
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select the **Default** runtime group and follow these steps:
```

Comment:
Is there a reason you chose `Default` runtime group here? In general, there's no need to specify a particular runtime group.